code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -185,9 +185,11 @@ var/list/gaslist_cache = null
 			if(return_temperature() < 50)
 				if(holder)
 					holder.freon_gas_act()
-			else
-				temperature -= 10
-
+	if(cached_gases["co2"] && cached_gases["freon"])
+		if(cached_gases["co2"][MOLES] > 1 && cached_gases["freon"][MOLES] > 1)
+			cached_gases["freon"][MOLES] -= 1
+			cached_gases["co2"][MOLES] -= 1
+			temperature += 2

```

Comment:
As discussed in coderbus. A method of quickly removing freon and reheating the room somewhat.